internal/pkg/state/registry/registry_test.go

```diff
@@ -301,3 +318,126 @@ func newTestState(t *testing.T, paths *knownpaths.Paths) *Registry {
 
 	return registry
 }
+
+func Test_parseIgnoredPatterns(t *testing.T) {
+	t.Parallel()
+	type args struct {
+		content string
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+		{
+			name: "with comments",
+			args: args{
+				content: "##dataapps\nkeboola.data-apps/12345\n##dataapps2\nkeboola.data-apps/123/rowId123\n##dataaps3\nkeboola.data-apps/12",
+			},
+			want: []string{
+				"keboola.data-apps/12345",
+				"keboola.data-apps/123/rowId123",
+				"keboola.data-apps/12",
+			},
+		},
+		{
+			name: "with empty lines",
+			args: args{
+				content: "##dataapps\n\n\n\n\n\nkeboola.data-apps/12345\n\t##dataapps2\nkeboola.data-apps/123\n##dataaps3\nkeboola.data-apps/12",
+			},
+			want: []string{
+				"keboola.data-apps/12345",
+				"keboola.data-apps/123",
+				"keboola.data-apps/12",
+			},
+		},
+		{
+			name: "empty file",
+			args: args{
+				content: "\n",
+			},
+			want: nil,
+		},
+		{
+			name: "error",
+			args: args{
+				content: "##dataapps\n\n\n\n\n\nkeboola.data-apps/12345\n\t##dataapps2\nkeboola.data-apps/123\n##dataaps3\nkeboola.data-apps/12",
+			},
+			want: []string{
+				"keboola.data-apps/12345",
+				"keboola.data-apps/123",
+				"keboola.data-apps/12",
+			},
+		},
+		{
+			name: "empty file",
+			args: args{
+				content: "\n",
+			},
+			want: nil,
+		},
+	}
+
+	for _, tt := range tests {
+		got := parseIgnoredPatterns(tt.args.content)
+		assert.Equal(t, tt.want, got)
+	}
+}
+
+func TestRegistry_applyIgnoredPatterns(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	registry := newTestState(t, knownpaths.NewNop(ctx))
+
+	type args struct {
+		pattern string
+	}
+
+	tests := []struct {
+		name    string
+		args    args
+		wantErr assert.ErrorAssertionFunc
+	}{
+		{
+			name: "empty patterns",
+			args: args{
+				pattern: "",
+			},
+			wantErr: assert.Error,
+		},
+		{
+			name: "wrong pattern",
+			args: args{
+				pattern: "wrong pattern",
+			},
+			wantErr: assert.Error,
+		},
+		{
+			name: "too long pattern",
```

Comment:
This looks like ok pattern, just changed dir structure. Could be changed in future

Reply:
I will update documentation: